src/utils.test.ts

```diff
@@ -39,6 +39,19 @@ namespace TypeChecks {
   type test30 = Expect<Equal<Subject.IsSpecial<' '>, false>>
   type test31 = Expect<Equal<Subject.IsSpecial<'*'>, true>>
   type test32 = Expect<Equal<Subject.IsSpecial<'_'>, false>>
+
+  type test33 = Expect<Equal<Subject.Truncate<'Hello, world', 9>, 'Hello,...'>>
```

Comment:
Like the words namespace, could you also create a namespace for these tests?

Reply:
fixed in [96f0483](https://github.com/gustavoguichard/string-ts/pull/41/commits/96f048307411594dfc3bb170f23f642f5d519e42)